jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/Vessel.java

```diff
@@ -20,10 +23,38 @@
 @AllArgsConstructor
 @Setter(AccessLevel.PRIVATE)
 @Entity
-@Table(name = "")
+@Table(name = "vessel")
 public class Vessel {
```

Comment:
Missing a nullable `validUntil` ZonedDateTime field (implementation defined field).  

Note the entity in JIT probably did not have it because it was added after the JIT 1.1 release.

Reply:
The IM that JIT is using (which looks pretty recent) does not have such a field

Reply:
Fine.